tests/selenium/test_selenium_appeal_generation.py

```
@@ -462,17 +462,17 @@ def test_session_scoped_localstorage_does_not_mix_appeals(self):
-        deadline = time.time() + 2
-        health_value = self.get_value("textarea#health_history")
-        while time.time() < deadline:
-            health_value = self.get_value("textarea#health_history")
-            if health_value == first_health:
-                break  # Restoration happened – fall through to the assertion
-            time.sleep(0.2)
+        # pending.  Use WebDriverWait to poll for a bounded time so that if
+        # the script is going to restore the old value we catch it, rather
+        # than asserting before the script has had a chance to run.
+        try:
+            WebDriverWait(self.driver, 2).until(
```

Ruff (0.15.5)

[error] `WebDriverWait` may be undefined, or defined from star imports

(F405)